files/jquery.prettycheckable/update.json

```diff
@@ -0,0 +1,8 @@
+{
+  "packageManager": "github",
+  "name": "prettyCheckable",
+  "repo": "arthurgouveia/prettyCheckable",
+  "files": {
+    "include": ["./img/*", "./dist/*", "./dev/*"]
```

Comment:
There is no point having a folder called `dist/` in production, thats why I want to remove it.
Is there any chance to ask the author to fix the image path? 

Regarding sprites, if they are not needed then you can exclude them `img/sprites/*`